modules/@apostrophecms/ui/ui/apos/components/AposTreeRows.vue

```diff
@@ -373,10 +373,11 @@ export default {
       }
 
       // Original default of just printing the row property value
-      if (row[col.cellValue]) {
-        return row[col.cellValue];
+      const publishedDoc = row._publishedDoc;
+      const value = (publishedDoc && (publishedDoc[col.cellValue] !== undefined) && publishedDoc[col.cellValue]) || row[col.cellValue];
```

Comment:
Again, we should be only showing one doc across the row.

Reply:
We are, not sure what this comment refers to.

Reply:
We're potentially showing data from both the published and the draft documents.